docs/source/reference/core/function.rst

```diff
@@ -11,3 +11,13 @@ Function
    chainer.force_backprop_mode
    chainer.no_backprop_mode
    chainer.grad
+
+Wrapper Functions
+=================
+
+Wrapper functions are backward-able functions (probably differentiable), that take an array or a Chainer :class:`~chainer.Variable` as an argument and return a Chainer :class:`~chainer.Variable`, or a tuple of a Chainer :class:`~chainer.Variable`.
```

Comment:
* How about
"Wrapper functions are backward-able (probably differentiable) plain Python functions that take..."
I mean, just saying "functions" is ambiguous.

* Description of inputs/outputs is duplicated with the following paragraphs. Can we simplify it?